src/crewai/cli/create_crew.py

```
@@ -1,80 +1,124 @@
-    templates_dir = package_dir / "templates"
-
-    # List of template files to copy
-    root_template_files = [
-        ".gitignore",
-        "pyproject.toml",
-        "README.md",
-    ]
-    tools_template_files = ["tools/custom_tool.py", "tools/__init__.py"]
-    config_template_files = ["config/agents.yaml", "config/tasks.yaml"]
-    src_template_files = ["__init__.py", "main.py", "crew.py"]
+    templates_dir = Path(str(package_dir) + "/templates" if not template_path else template_path)
```

Ruff (0.8.0)

Use `template_path if template_path else str(package_dir) + "/templates"` instead of `str(package_dir) + "/templates" if not template_path else template_path`

Replace with `template_path if template_path else str(package_dir) + "/templates"`

(SIM212)